apps/arena_load_test/lib/arena_load_test/socket_handler.ex

```diff
@@ -35,12 +35,22 @@ defmodule ArenaLoadTest.SocketHandler do
     game_id = Serialization.GameState.decode(game_state).game_id
     Logger.info("Client joining game with id: #{game_id}")
 
+    case :ets.lookup(:clients, state.client_id) do
+      [{client_id, _}] ->
+        :ets.delete(:clients, client_id)
+
+      [] ->
+        raise KeyError, message: "Client with ID #{state.client_id} doesn't exist."
```

Comment:
More a curiosity, why raise?

Reply:
Because I want to know when I'm messing up with the clients and players tables.
They won't raise unless I add or delete them wrongly, or something unexpected happens (a future worry).